### PR TITLE
Remove healthCheckHTTPPortWarning

### PR DESCRIPTION
**Pull request by pradeepnnv**
Fixes part of https://github.com/jaegertracing/jaeger/issues/2738

Signed-off-by: pradeepnnv <pradeepnnv@gmail.com>

### CODE REVIEW

File: cmd/flags/admin.go [hunk 1]
```diff
@@ -36,8 +36,7 @@ const (
 	adminHTTPPort       = "admin-http-port"
 	adminHTTPHostPort   = "admin.http.host-port"
 
-	healthCheckHTTPPortWarning = "(deprecated, will be removed after 2020-03-15 or in release v1.19.0, whichever is later)"
-	adminHTTPPortWarning       = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
+	adminHTTPPortWarning = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
```
Comment (yurishkuro) [inline cmd/flags/admin.go:39]: please remove both deprecated flags since we're changing this module

**Review by yurishkuro — COMMENTED** [review]
please add a breaking change note to the CHANGELOG

File: cmd/flags/admin.go (hunk 1 above)
Comment (pradeepnnv) [reply cmd/flags/admin.go]: Removed adminHTTPPortWarning as well. Please review.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ Changes by Version
 
 #### Breaking changes
 
+* Remove depricated flags `healthCheckHTTPPortWarning` & `adminHTTPPortWarning` ([#2752](https://github.com/jaegertracing/jaeger/pull/2752), [@pradeepnnv](https://github.com/pradeepnnv))
```
Comment (yurishkuro) [inline CHANGELOG.md:11]: please list the actual CLI flags, not the variable names in the code

Comment (yurishkuro) [reply CHANGELOG.md:11]:
```suggestion
* Remove deprecated flags `--health-check-http-port` & `--admin-http-port`, please use `--admin.http.host-port` ([#2752](https://github.com/jaegertracing/jaeger/pull/2752), [@pradeepnnv](https://github.com/pradeepnnv))
```

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
@pradeepnnv Thank you!

**Comment by pradeepnnv** [comment]
Thanks @yurishkuro for your help and guidance. Hope i'll be able to contribute more for Jaeger.

**Pull request merged**